### PR TITLE
updated all files

### PR DESCRIPTION
**Pull request by francis3400**
- [ ] updated index files
- [ ] updated style files
- [ ] Created a pull request
- [ ] Corrected all errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,7 +55,9 @@ <h1 class="logo">My Logo</h1>
             </li>
           </ul>
         </div>
+
```
Comment (christianbiring1) [inline index.html:58]: - [ ] Kindly make sure your social icons are well aligned as you can see :point_down: 
image

File: index.html [hunk 2]
```diff
@@ -77,19 +78,21 @@ <h1 class="logo">My Logo</h1>
...
-              <button type="button">css</button>
-              <button type="button">javascript</button>
-            </div>
+          
+              
+             <ul  class="htmlButtons" >
+              <li><button type="button">html</button></li>
+              <li><button type="button">css</button></li>
+              <li><button type="button">javascript</button></li>
+              </ul>
```
Comment (christianbiring1) [inline index.html:87]: - [ ] Kindly make sure you stick to the design :point_down: 
image
The Figma template :point_down: 
image

File: index.html [hunk 3]
```diff
@@ -180,18 +192,18 @@ <h1 class="logo">My Logo</h1>
...
+            <p id="aboutText">
               Hello I’m a software developer! I can help you build a product ,
               feature or website Look through some of my work and experience! If
               you like what you see and have a project you need coded, don’t
               hestiate to contact me.
             </p>
-          </div>
-         <div class="landingH3 landingtxt"><h3>LET’S CONNECT</h3></div>
+        
+         <h3 class="connect">LET’S CONNECT</h3>
```
Comment (christianbiring1) [inline index.html]: - [ ] Kindly make sure you stick to the design. Your design :point_down: 
image
Figma design :point_down: 
image

File: index.html [hunk 4]
```diff
@@ -180,18 +192,18 @@ <h1 class="logo">My Logo</h1>
...
-          <div class="hello" id="hello2">
-            <h2>About Myself</h2>
-          </div>
-          <div>
-            <p class="landingText" id="aboutText">
+          
+            <h2 class="hello2">About <br> Myself</h2>
+        
+        
+            <p id="aboutText">
```
Comment (christianbiring1) [inline index.html:199]: Your design :point_down: 
image
 Figma design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @francis3400 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
164625215-44f24c98-53c8-49c1-b68f-05723a584dc2

## Highlights
- [ ] All linters are green :heavy_check_mark: 
- [ ] Descriptive PR.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — COMMENTED** [review]
Hi @francis3400, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a professional 👀 README file according with the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by emmiiorji — APPROVED** [review]
Hi @francis3400, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations


## Highlights✔

- You have implemented suggested changes
- Linter checks are passing
- Good commit messages
- Adherence to GitHub flow


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**